lib/analytics.js

```diff
@@ -335,10 +335,11 @@ Analytics.prototype.track = function(event, properties, options, fn) {
  * @param {Element|Array} links
  * @param {string|Function} event
  * @param {Object|Function} properties (optional)
+ * @param {Object|Function} options (optional)
```

Comment:
Can options be a function? Weird that properties can be a function...

Reply:
Not sure what the use case is, but if we allow event and properties be a function might as well stay consistent and let options be a function.
> https://segment.com/docs/sources/website/analytics.js/#track-link

Reply:
Does the code work if `options` is a function? If so - we should add a test case for that.

Reply:
Yes, it works with an options function. There is a test case for the options function already added.

Reply:
Ah thanks - I see the test!